src/readwrite.rs

```diff
@@ -71,6 +71,13 @@ impl<'env, K> Writer<'env, K> where K: AsRef<[u8]> {
             .map_err(StoreError::LmdbError)
     }
 
+    // TODO: duplicate data
```

Comment:
I think you're referring to `MDB_DUPSORT` support, yes? I think it's OK to make that a separate method, perhaps `remove_value`. You could even stub it out with `unimplemented!()`.

Even better would be to make this a method _only on a dupsort store_ — it would need a little bit of reorganizing of types and traits, but when I see "If the database does not support sorted duplicate data items (MDB_DUPSORT) the data parameter is ignored" in the docs, I see a footgun that we can avoid by using the type system.

Reply:
Yes, that's what I was referring to. I've replaced that comment in e973afb with an unimplemented!() function (and a test for it), with your "even better" variant as a code comment to prompt the implementer.